src/modules/launcher/Wox.Core/Plugin/PluginConfig.cs

```diff
@@ -81,7 +83,9 @@ private static PluginMetadata GetPluginMetadata(string pluginDirectory)
                 // for plugin still use old ActionKeyword
                 metadata.ActionKeyword = metadata.ActionKeywords?[0];
             }
+#pragma warning disable CA1031 // Do not catch general exception types
```

Comment:
It would be better to use attributes to suppress this error. You can find an example here: https://github.com/microsoft/PowerToys/pull/6958/files#diff-82046de380447e40c4e6c040bdb9643d4f4c019d934ef9c310eba7920891b27aR333